docs/upgrade.md

```diff
@@ -45,6 +45,20 @@ for detail on upgrading to Marionette 3. This technique works in both Marionette
 
 The `childEvents` attribute was renamed to `childViewEvents`.
 
+Child view events now only bubble a single level to their direct parent. To
```

Comment:
Actually this unfortunately isn't true.  multi-level bubbling still occurs in some instances, although this was never meant to happen.